common/prettify.pm

```diff
@@ -1755,7 +1755,8 @@ sub Test_Handler ($)
         || $s =~ m/is not recognized as an internal or external command/
         || $s =~ m/can't open input/
         || $s =~ m/ACE_SSL .+ error code\: [0-9]+ - error\:[0-9]+\:SSL routines\:SSL3_READ_BYTES\:sslv3 alert certificate expired/
-        || $s =~ m/memPartFree: invalid block/ )
+        || $s =~ m/memPartFree: invalid block/
+        || $s =~ m/: ThreadSanitizer: /)
```

Comment:
Not a big deal here, but you might want to add the entry before the last one so that the `memPartFree` line isn't changed by this PR.